scripts/functions.sh

```diff
@@ -297,6 +297,17 @@ function copyOutput() {
   echo "Copying collector output files..."
   mkdir -p ./output
   ${K8S_CLI_BIN} cp -n "$namespace" collector:output ./output
+  flowFile=$(find ./output -name "*json" | sort | tail -1)
+  flowFileName=$(basename "$flowFile")
+  TMP_FLOWFILE="/$MANIFEST_OUTPUT_PATH/$flowFileName"
+  cp "$flowFile" "$TMP_FLOWFILE"
+  UPDATED_JSON_FILE="/$MANIFEST_OUTPUT_PATH/updated_$flowFileName"
+  echo "[" >> "$UPDATED_JSON_FILE"
+  # remove last line and "," (last character) of the last flowlog for valid json
+  sed '$d' "$TMP_FLOWFILE" | sed '$ s/.$//' >>  "$UPDATED_JSON_FILE"
+  echo "]" >> "$UPDATED_JSON_FILE"
```

Comment:
The linter is suggesting a nice change here:
```
In ./scripts/functions.sh line 305:
  echo "[" >> "$UPDATED_JSON_FILE"
  ^-- SC2129 (style): Consider using { cmd1; cmd2; } >> file instead of individual redirects.
```

Which would result in:
```suggestion
  { 
    echo "["
    # remove last line and "," (last character) of the last flowlog for valid json
    sed '$d' "$TMP_FLOWFILE" | sed '$ s/.$//'
    echo "]"
  } >> "$UPDATED_JSON_FILE"
```

Reply:
thanks, indeed that's more elegant.